### ISSUE TITLE
Request: Show modifier keys on the left when aligned to the right

### ISSUE THREAD

**Issue by jlampel** [nutti/Screencast-Keys#107]
**Description about the feature**  
The best place for me to put Screencast Keys is the bottom right of the viewport, since it's out of the way and doesn't block the redo panel. But, when a modifier key is pressed, it jumps the mouse around and is hard to follow. If the feature is aligned to the right, could the modifier keys appear to the left? 

**Are you willing to contribute about this feature. (Yes/No)**  
Yes, I wouldn't mind taking a crack at it if it's a simple enough fix. 

**Screenshots** [Optional] 
![blender_TfzVkT0tKa](https://user-images.githubusercontent.com/12982300/159570377-436c79b4-9305-4505-848b-c1627f6af15c.gif)

**Comment by nutti** [comment]
@jlampel 

This feature looks good to me.

You need to fix below two functions.

* https://github.com/nutti/Screencast-Keys/blob/276edd874efa93755e15e36506cb23d784cdcf88/src/screencast_keys/ops.py#L855
* https://github.com/nutti/Screencast-Keys/blob/276edd874efa93755e15e36506cb23d784cdcf88/src/screencast_keys/ops.py#L1118

If you think this fixes are difficult, I will tackle this issue.
What do you think?

**Comment by nutti** [comment]
@jlampel 

Thanks.
If you still want to contribute this feature, I will wait.

It is no problem that you quit to tackle this issue.
In the case, I will tackle this issue instead.

**Comment by nutti** [comment]
This request is now in the master branch.
Thanks for the request.
